samples/documentation/lulesh_sample1_unix_commented.yaml

```diff
@@ -0,0 +1,234 @@
+          # the top level directory that contains each parameterized workspace.
+          # In this case, it points to './sample_output/lulesh/run-lulesh'.
+          cmd: |
+            echo "Unparameterized step with Parameter Independent dependencies." >> out.log
+            echo $(run-lulesh.workspace) > out.log
+            ls $(run-lulesh.workspace) > ls.log
+          # The 'post-process-lulesh' step is dependent on the completion of
+          # all parameterizations of 'run-lulesh' as represented by the '_*'.
+          # Because this step is not parameterized, it is a singular step that
+          # waits for all 'run-lulesh' steps to complete.
+          depends: [run-lulesh_*]
+
+    - name: post-process-lulesh-trials
+      description: Post process all LULESH results.
+      run:
+          cmd: |
+            echo "Parameterized step that has Parameter Independent dependencies" >> out.log
+            echo "TRIAL = $(TRIAL)" >> out.log
+            echo $(run-lulesh.workspace) >> out.log
+            ls $(run-lulesh.workspace) > out.log
+          # 'post-process-lulesh-trials' like 'post-process-lulesh' is dependent
+          # on all parameterizations of 'run-lulesh'. In this case, because this
+          # step is parameterized based on $(TRIALS), there will be a version of
+          # 'post-process-lulesh-trials' for each unique value of $(TRIAL) that
+          # each will wait on all combinations of 'run-lulesh'.
+          depends: [run-lulesh_*]
+
+    - name: post-process-lulesh-size
+      description: Post process all LULESH results.
+      run:
+          cmd: |
+            echo "Parameterized step that has Parameter Independent dependencies" >> out.log
+            echo "SIZE = $(SIZE)" >> out.log
+            echo $(run-lulesh.workspace) >> out.log
+            ls $(run-lulesh.workspace) | grep $(SIZE.label) >> out.log
+          # This step is more or less exactly the same as 'post-process-lulesh-trials'
+          # but it is parameterized on $(SIZE). This step is meant to illustrate
+          # that parameters can be reused for filtering or other useful things when
+          # dependent on parameterizations of other steps.
+          depends: [run-lulesh_*]
+
+      # NOTE: The 'depends' entry in a specification does not use parameterized
+      # names -- instead Maestro attempts to infer the expansion of the graph
+      # based on context via looking to see if a step uses parameters, workspaces,
+      # and if the step depends on other steps that are parameterized.
+      # NOTE: Second note -- this example lacks a type of step. It is possible
+      # to have a step that is expanded because its dependencies have been
+      # expanded. For example, if you wanted to run individual post processing
+      # after each 'run-lulesh' step, you might have a step called
+      # 'post-process-run' that is dependent on 'run-lulesh'. This notation

```

Comment:
This should explicitly say, "the job step specification would then contain the value 'depends : [run_lulesh]'."

If this is what you mean.